yarn.lock

```diff
@@ -14344,9 +14344,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#87c9c31980e8dffc45b388d7af5f7ea5b6800a1f":
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#baa5f1ea73c9183398eedbe40b40c5695dbce77c":
```

Comment:
Looks like the `package.json` was updated at some point, but the yarn.lock file never was.